wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,105 @@
+---
+tags:
+  - development
+  - osu!dev
+---
+
 # Discord servers
 
-Index page for all articles regarding community [Discord](https://discord.com/) servers.
+As the community grows and expands, several [Discord](https://discord.com/) servers have been made and maintained by the community to specialise in the game's many different facets.
+
+Each server provides a space with a specific purpose in mind, where players get together to discuss and work on various different projects. This index aims to serve as an index where users can find a subcommunity they can best fit in.
+
+## osu! Development server
```

Comment:
```suggestion
## Official osu! server
```
it's not called "osu! Development" anywhere, also maybe add "Official" like this so its extrasuperobvious idk